### DISCUSSION TITLE
/about-the-author/csdn-1000wan.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#67] [Announcements]
# /about-the-author/csdn-1000wan.html

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/about-the-author/csdn-1000wan.html

**Comment by BY-H** [comment]
二哥牛逼，2022 我们一起飞

**Comment by Hehebulu** [comment]
很喜欢王哥总结的这些东西,对我来说就是宝贵的财富,加油王哥,我支持你
